src/libs/ReportUtils.ts

```diff
@@ -9787,6 +9787,10 @@ function canLeaveChat(report: OnyxEntry<Report>, policy: OnyxEntry<Policy>, isRe
         return true;
     }
 
+    if (isMoneyRequestReport(report) && currentUserAccountID !== report?.managerID && currentUserAccountID !== report?.ownerAccountID && !isPolicyAdminPolicyUtils(policy)) {
```

Comment:
Why is PolicyAdmin included in this check? We should be using the checks from [here](https://github.com/Expensify/App/blob/main/README.md#reports)

Reply:
I see what you're saying. Thanks for the explanation